### PR TITLE
Populate my events

### PR DESCRIPTION
**Pull request by thecoldmoon**
Links DB with front-end

### CODE REVIEW

File: src/Components/EventCard.js [hunk 1]
```diff
@@ -32,8 +35,10 @@ const EventCard = (props) => {
                 return Images.events.sports;    
             case "THEATRE":
                 return Images.events.theatre;
-            default:
+            case "":
                 return Images.events.coffee;
+            default:
+                return lowerType;
```
Comment (MickeyDang) [inline src/Components/EventCard.js]: Why change the default? Couldn't `lowerType` be nothing in this case which would render a broken image?

File: src/Components/MyEventPage.js [hunk 2]
```diff
@@ -65,88 +28,102 @@ class MyEventPage extends Component {
...
-        );
-    }
-}
-
-export default MyEventPage
\ No newline at end of file
+                </div>
+        </div>
+    );
+}
```
Comment (MickeyDang) [inline src/Components/MyEventPage.js]: Add new line

**Review by MickeyDang — COMMENTED** [review]
I think its pree solid refactor but I'd rather Jackson look because he has more context. I didn't see anything bad though. Just left some questions

@zhengwin

**Review by zhengwin — APPROVED** [review]
LGTM

**Pull request merged**